screens/CheckoutScreen.js

```diff
@@ -181,61 +184,113 @@ export default class CheckoutScreen extends React.Component {
-              <ScrollView>
-                {Object.entries(cart).map(([id, product]) => {
-                  return (
-                    product.quantity > 0 && (
-                      <QuantityModal key={id} product={product} isLineItem callback={this.updateQuantityCallback} />
-                    )
-                  );
-                })}
-              </ScrollView>
-            </View>
-            <RewardModal
-              totalBalance={totalBalance}
-              customer={customer}
-              rewardsAvailable={this.state.rewardsAvailable}
-              rewardsApplied={this.state.rewardsApplied}
-              callback={this.applyRewardsCallback}
-            />
-            <Text
+            <View
               style={{
-                fontWeight: 'bold',
-                textAlign: 'center'
+                paddingTop: 13,
+                paddingLeft: 14,
+                paddingRight: 14,
+                paddingBottom: 0,
+                flexDirection: 'column',
+                justifyContent: 'space-between',
+                flex: 1
               }}>
-              Order Total ${totalSale.toFixed(2)}
-            </Text>
-            <TouchableOpacity onPress={() => this.handleSubmit()}>
-              <TextHeader style={{ color: '#008550' }}>Checkout</TextHeader>
-            </TouchableOpacity>
+              <View style={{ height: '60%' }}>
+                <Subhead>Current Sale</Subhead>
+                {/* Cart container */}
+                <View style={{ height: '100%', paddingBottom: '5%' }}>
+                  <ScrollView>
+                    {Object.entries(cart).map(([id, product]) => {
+                      return (
+                        product.quantity > 0 && (
+                          <QuantityModal key={id} product={product} isLineItem callback={this.updateQuantityCallback} />
+                        )
+                      );
+                    })}
+                  </ScrollView>
+                </View>
+              </View>
+              {/* Should be greyed out if totalPrice < 5 */}
+              <View>
+                <RewardModal
+                  totalBalance={totalBalance}
+                  customer={customer}
+                  rewardsAvailable={this.state.rewardsAvailable}
+                  rewardsApplied={this.state.rewardsApplied}
+                  callback={this.applyRewardsCallback}
+                />
+                {/* When different types of rewards are created, we can add rewards amount to state. For now, rewards
+              amount is equal to rewards applied * 5. */}
+                <SubtotalCard subtotalPrice={subtotal.toFixed(2)} rewardsAmount={this.state.rewardsApplied * 5} />
+                <TotalCard totalPrice={totalSale.toFixed(2)} totalPoints={pointsEarned} />
+              </View>
+            </View>
+            <FilledButtonContainer onPress={() => this.handleSubmit()}>
+              <ButtonLabel>Complete Purchase</ButtonLabel>
```

Comment:
Design update: Ace wanted to keep the language consistent across the app (since it is currently 'Current Sale' in the heading, 'Complete Purchase' on the button, 'Confirm Transaction' on the system dialog, then 'Purchase Summary' on the confirmation), so can we change everything to Sale? 

Complete Purchase -> Complete Sale
Purchase Summary -> Sale Summary
Confirm Transaction -> Confirm Sale

Reply:
This is a recent update but should be reflected in Figma